state/userAccount.go

```diff
@@ -31,10 +33,21 @@ func NewEmptyUserAccount() *userAccount {
 }
 
 // NewUserAccount creates new simple account wrapper for an AccountContainer (that has just been initialized)
-func NewUserAccount(address []byte, hasher hashing.Hasher, marshaller marshal.Marshalizer) (*userAccount, error) {
+func NewUserAccount(
+	address []byte,
```

Comment:
maybe this is more relevant for a future refactor, but i think having an args struct would be better, at least when we have more params; also in other cases

Reply:
Refactored.